### PR TITLE
ENH: add helpers with fzf/jq and a try_me script

### LINKED ISSUE #5: Add fzf integration
* We have fzf installed in our environment
* On applicable machines, install the bindings
* And also use `fzf` to interactively grep results in say `find_ioc`

### PR DESCRIPTION
**Pull request by klauer**
* Try these scripts without installing them (it'll mostly work, at least)
   * `bash --rcfile try_me.sh`
* Fuzzy find IOCs (`grep_ioc`)
* Fuzzy find an IOC and then `cd` to its data directory  (`iocdata`)
* Fuzzy find an IOC and then `cd` to its startup directory (`cdioc`)
* Add helpers for jq/fzf to find them in a released pcds-env 
* Enable fzf bindings for ctrl-r and ctrl-t in bash

Closes #5

### CODE REVIEW

File: helpers.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#
+# This file contains some helper scripts for your dotfiles to reuse.
+#
+
+if [ -z "$_PCDS_CONDA_FOR_UTILS" ]; then
+    _PCDS_CONDA_FOR_UTILS=/cds/group/pcds/pyps/conda/py39/envs/pcds-5.3.0/
+fi
+
+# ALL_IOCS_TEXT=/cds/data/iocData/.all_iocs/iocs.txt
+_ALL_IOCS_JSON=/cds/data/iocData/.all_iocs/iocs.json
```
Comment (tangkong) [inline helpers.sh:11]: You have a script somewhere to generate this json, I assume?  What happens when this falls out of date?

Comment (klauer) [reply helpers.sh:11]: Yes, it's a periodic (15 minutes, IIRC) personal cron job on psbuild-rhel7 that updates that file by way of:
https://github.com/klauer/pscron/blob/master/update_all_iocs.sh (effectively a call to `whatrecord` to use its ioc manager config parsing tools)
It also generates: https://confluence.slac.stanford.edu/display/PCDS/EPICS+IOCs+Deployed+in+IOC+Manager
And commits locally to that `.all_iocs` git repo.

If it falls out of date, I get a bunch of e-mails indicating my cron job failed, annoying me to update it.
If I don't pay attention to it, then everyone's IOC list will be stale until I get back.

**Review by tangkong — APPROVED** [review]
I think this is super slick.  I'm learning a lot from these dotfiles.  

I was thinking that as someone who doesn't know what a lot of the functionality is, having little `try-module.sh` scripts could be neat.  It might help ease people into the aliases and such.  That seems like a lot of work though, maybe for a future time.

**Review by ZLLentz — APPROVED** [review]
This is well beyond what I've set up for myself- I need to set aside some time to try everything out

**Comment by tangkong** [comment]
I'm not sure how exactly I'd split it up, but it would initialize subsets of the dotfiles functionality for someone to try.  I don't think they would end up being very long script files, maybe one initializes the bash aliases, one initializes engineering tool shortcuts, etc.  

Maybe with a little print at the top summarizing what got loaded, people could try aliases without actually "installing" the dotfiles.  I'll think a bit more about it, maybe make a PR if get around to playing with these.

**Comment by klauer** [comment]
Interesting idea - like a tour of the different scripts. It'd be really cool to have a tutorial mode to walk through the individual functions and all.

**Pull request merged**